pkg/dependency/credhelper/cred_helper_binary.go

```diff
@@ -103,6 +103,38 @@ func updateConfigFile(bin *credhelperbin) error {
 	return nil
 }
 
+func (bin *credhelperbin) binaryInstalled() (bool, error) {
+	return binaryInstalled(bin)
+}
+
+func (bin *credhelperbin) configFileInstalled() (bool, error) {
+	cfgPath := fmt.Sprintf("%s%s", bin.hcfg.finchPath, "config.json")
```

Comment:
Does finch path include the OS separator already?

Reply:
Yes

Reply:
maybe it's simpler to use `filepath.Join` to keep it consistent with how we construct filepaths